internal/cli/atlas/metrics/disks/list.go

```diff
@@ -57,14 +58,21 @@ var listTemplate = `{{range .Results}}
 {{.PartitionName}}{{end}}
 `
 
-// mongocli atlas metric(s) process(es) disks lists <hostname:port>.
-func DisksListBuilder() *cobra.Command {
-	opts := &DisksListsOpts{}
+// mongocli atlas metric(s) disks lists <hostname:port>.
+func ListBuilder() *cobra.Command {
+	opts := &ListsOpts{}
 	cmd := &cobra.Command{
 		Use:     "list <hostname:port>",
 		Short:   "List available disks for a given host.",
 		Aliases: []string{"ls"},
 		Args:    require.ExactArgs(1),
+		Annotations: map[string]string{
+			"args":              "hostname:port",
+			"requiredArgs":      "hostname:port",
+			"hostname:portDesc": "Hostname and port number of the machine running the Atlas MongoDB process.",
```

Comment:
instance

Reply:
This is not done. Did we choose to go with machine, still? In the cloud world, the industry standard is to use "instance" in such cases, I think.